docs/integrations/amazon-aws/ses.md

```diff
@@ -258,7 +258,7 @@ This section provides examples of the Amazon SES App dashboards, along with desc
 
 See the overview of SES CloudTrail events including failed, and successful events, error codes, users, and event locations.
```

Comment:
```suggestion
See the overview of SES CloudTrail events including failed, successful events, error codes, users, and event locations.
```

Reply:
removing the comma, to say 'failed and successful events'